src/components/FIlters/Filter.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useContext, useState, useEffect } from "react";
+import arrowRight from "/home/vladimir/Desktop/Movies/movies/src/assets/arrow-right.svg";
+import {
+  FilterCont,
+  DropdownFilterBody,
+  DropdownFilter,
+  FilterTitle,
+  FilterContent,
+  Titile,
+  FilterItem,
+  SelectionBtn,
+  Selections,
+  SubmitBtn,
+} from "./Filter.style";
+import { MoviesContext } from "../../contexts/moviesContext";
```

Comment:
google how to use [aliases](https://medium.com/dailyjs/using-import-aliases-in-javascript-a0b46237601c). You already have the configs in your jsconfig.json